inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -869,15 +938,18 @@ total of `r nrow(linelist_cleaned)` cases. There were
 The most affected age group was `r tab_linelist(linelist_cleaned, age_group) %>% slice(which.max(n)) %>% pull(value)` years. 
 
 
-#### Demographics 
-
 
+#### Demographics 
 
 Cases by age group and definition 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+describe_by_age_group_and_def
 
+This chunk will create a table of cases by age group and case definition.
+You can only use this chunk if you have lab results and have defined a case
+definition variable.
```

Comment:
otherwise set this chunk to eval = FALSE

Reply:
as above re eval